x-pack/plugins/siem/public/pages/home/index.tsx

```diff
@@ -38,6 +31,22 @@ const WrappedByAutoSizer = styled.div`
   height: 100%;
 `;
 
+const gutterTimeline = '70px'; // Temporary until timeline is moved - MichaelMarcialis
+
+const Page = styled(EuiPage)`
+  padding: 0 ${gutterTimeline} ${euiLightVars.euiSizeL} ${euiLightVars.euiSizeL};
+`;
+
+const NavGlobal = styled.nav`
+  background: ${({ theme }) =>
+    theme.darkMode ? euiDarkVars.euiColorEmptyShade : euiLightVars.euiColorEmptyShade};
+  border-bottom: ${({ theme }) =>
+    theme.darkMode ? euiDarkVars.euiBorderThin : euiLightVars.euiBorderThin};
+  margin: 0 -${gutterTimeline} 0 -${euiLightVars.euiSizeL};
+  padding: ${euiLightVars.euiSize} ${gutterTimeline} ${euiLightVars.euiSize}
```

Comment:
Similar to the above you can use `props.theme.eui` to get rid of the `euiVars` imports.

Reply:
👍